proposals/lip-0048.md

```diff
@@ -25,15 +25,57 @@ This LIP defines the fee system in a modular way, as currently [used in the Lisk
 
-### Minimum Fee per Transaction
+### Transaction Fees
 
-As introduced in [LIP 0013][lip-0013], all transactions must have a fee greater or equal to a minimum fee (which can be zero). The minimum fee is computed from the transaction size. Chains can configure their minimum fee per byte, with a possibility to define a period of `MAX_BLOCK_HEIGHT_ZERO_FEE_PER_BYTE` blocks after the genesis block when the minimum fee per byte is set to zero.
+Each transaction in the Lisk ecosystem has a *transaction fee* associated with it, as defined in [LIP 0068][lip-0068]. Transaction fees prevent spamming a blockchain with unlimited transactions at zero cost. In addition, a part of the transaction fee is provided as a reward to the validators who operate the network and maintain its security, as we explain in detail below. 
 
-Chains can choose that this minimum fee gets either burned (to avoid that validators can send transactions in the blocks they generate without cost) or transferred to the specified pool address `ADDRESS_FEE_POOL`. The latter choice could be beneficial for sidechains, as fees collected in this way can be used as an incentive for validators and other protocol participants. In the Lisk mainchain, the minimum fee is always burned.
+#### Fee Token ID
+
+Each chain can configure the token used to pay fees, as defined by `TOKEN_ID_FEE`. On the Lisk mainchain, the token used for transaction fees is the LSK token.
+
+#### Minimum Fee per Transaction
+
+As introduced in [LIP 0013][lip-0013], all transactions must have a transaction fee that is greater than or equal to a *minimum fee* that is always deducted from the part of the fee that goes to the block generator. The minimum fee, unless it is zero, is intended to discourage validators from sending zero-cost transactions in the blocks they generate.
+
+For a transaction `trs`, the minimum fee is determined by its size and the proportionality constant `MIN_FEE_PER_BYTE`: 
+```python
+trsSize = length(encodeTransaction(trs))
+minFee = MIN_FEE_PER_BYTE * trsSize
+```
+Note that chains can freely configure their `MIN_FEE_PER_BYTE`, which can even be set to zero. The `MIN_FEE_PER_BYTE` value cannot be changed, unless a hard fork occurs. For this reason, we allow sidechains to define a period of `MAX_BLOCK_HEIGHT_ZERO_FEE_PER_BYTE` blocks after the genesis block, for which the minimum fee does not apply. 
+
+This could, for example, be useful for sidechains that want to use LSK as the fee token. Initially, there is no LSK on the sidechain, so the sidechain cannot process transactions unless `MIN_FEE_PER_BYTE` is set to zero, which is generally not desirable since it essentially disables the feature of minimum fee for the chain. By granting an initial exemption period of `MAX_BLOCK_HEIGHT_ZERO_FEE_PER_BYTE` blocks, users have a window of time to transfer LSK tokens to the sidechain, after which the initially set value of `MIN_FEE_PER_BYTE` would start being applied. 
+
+### Method `payFee`
```

Comment:
I would still try to give a different title here, such as "additional fees", or "state growth fees". The payFee is just the function that does this.

Reply:
I renamed the paragraph "Additional Fees", I think it's a general term which doesn't say much, but covers everything :)